doc/package-specific-notes.chapter.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Package-specific notes
```

Comment:
Oops, I haven't read the manual since forever (yo can only trust source code!), so I skimmed the chapter list and missed that.

Reply:
On the other hand, I think the package notes are a mess right now, as there is no separation between notes of type «this is how you update X.org» and «this is how you _use_ Steam»

Reply:
I can totally understand that, but the notes do have to go somewhere appropriate in the manual and it would be the best place their is currently. But a restructuring is needed as you said.

Reply:
Yes, I did miss the preexisting section initially.

Actually, maybe it should be split with package use notes going closer to the beginning and package editing notes staying after all the explanations of package writing?

Reply:
> Actually, maybe it should be split with package use notes going closer to the beginning and package editing notes staying after all the explanations of package writing?

Subsections like that could help, yes.

```
# Packages Notes:

## Using packages
[explanation/intro]

## Writing packages
[explanation/intro]
```
If I'm understanding correctly?

Reply:
No. «Using packages» should be _before_ the detailed walkthrough os `stdenv` _code_; «Updating speciric packages» should obviously be after all the generic explanations…